beekeeper-scheduler/src/test/java/com/expediagroup/beekeeper/scheduler/service/ExpiredHousekeepingMetadataSchedulerServiceTest.java

```diff
@@ -123,13 +120,9 @@ public void typicalUpdatePartitionedTableWithShorterCleanupDelay() {
     verify(existingTable).setHousekeepingStatus(metadata.getHousekeepingStatus());
     verify(existingTable).setCleanupDelay(metadata.getCleanupDelay());
     verify(existingTable).setClientId(metadata.getClientId());
-    verify(existingTable).getCleanupTimestamp();
```

Comment:
Why have you removed these verifications in the scheduler?

Reply:
Because when implementing Lombok, some of these verifications stopped working. I asked you what to do about them and you said that it was fine to delete them